src/transport/http2.rs

```diff
@@ -407,7 +428,7 @@ impl<Out: RpcMessage> SendSink<Out> {
         }
     }
     fn serialize(&self, item: Out) -> Result<Bytes, SendError> {
-        let mut data = Vec::new();
+        let mut data = Vec::with_capacity(1024);
```

Comment:
I'm used to seeing 8KiB buffers for this kind of think rather than just one.  Would there be any negative effect when making this a bit bigger right away?

Reply:
This is not a buffer that gets kept around, but a thing that later gets converted into a Bytes for hyper. So I figured smaller is better. I guess it depends on the average msg size.

Quite a few inefficiencies in this entire thing if you look closely. So much copying, resizing, .... But still you get multiple gigabytes per sec, so it is not going to be the limiting factor for anything.